core/common/src/test/scala/org/typelevel/otel4s/TestInstances.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2022 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.typelevel.otel4s
+
+import cats._
+import cats.laws.discipline._
+import cats.syntax.all._
+
+object TestInstances {
+  implicit def eqForTextMapGetter[A: ExhaustiveCheck]: Eq[TextMapGetter[A]] =
```

Comment:
maybe this is a style choice, but I feel like this could just go in whatever file tests the laws for `TextMapGetter`

Reply:
That's fine with me.  If we need it in two places, we can always extract.